File: Backend/quiz_craft/models/game.py
from django.db import models
from .task import Task


class Game(models.Model):
    GAME_TYPES = [
        ('SINGLEANSWER', 'Single Answer'),
    ]

    WEAPONS = [
        ('SWORD', 'Sword'),
        ('HAMMER', 'Hammer'),
        ('SCYTHE', 'Scythe'),
    ]

    type = models.CharField(max_length=20, choices=GAME_TYPES)
    min_win_grade = models.IntegerField()
    player_weapon = models.CharField(max_length=20, choices=WEAPONS)
    enemy_weapon = models.CharField(max_length=20, choices=WEAPONS)
    task = models.ForeignKey(Task, on_delete=models.CASCADE)

File: Games/SingleAnswerGame/Assets/Scripts/Global/Data/Game.cs
using Newtonsoft.Json;

namespace Global.Data
{
    public class Game
    {
        public string Type { get; set; }
        
        [JsonProperty("min_win_grade")]
        public int MinWinGrade { get; set; }
        
        [JsonProperty("player_weapon")]
        public string PlayerWeapon { get; set; }
        
        [JsonProperty("enemy_weapon")]
        public string EnemyWeapon { get; set; }
        
        public Task Task { get; set; }
    }
}